v4/source/users.yaml

```diff
@@ -128,6 +128,36 @@
         '401':
           $ref: '#/responses/Unauthorized'
 
+  /users/names:
```

Comment:
I know it's kind of redundant but I think we should use the actual field name so `users/usernames`. I think we also use `username` for the get by username endpoint